Initial review of iocage.8

Mrt134 opened the pull request:
- Added Warren Block and Tim Moore to the author list.
First pass SYNOPSIS review
- Added missing "clone" command.
- "console": remove --force flag.
- "create": add --name flag, remove obsolete commands.
- "df": Add _long, add --sort flags.
- "exec": remove ALL  argument.
- "fetch": add _file, --name, --accept, --noaccept, remove obsolete optional commands.
- "fstab": add ACTION arguments to flags, rework lines.
- "get": rework lines, add _all, add _pool
- "import": remove Jail_value line
- "list": add dataset_type arguments, add --sort flags, add _long flag
- "pkg": remove JAIL arg, replace with UUID | TAG
- "rollback": move jail argument, remove obsolete argument.
- "set": rework  arguments
- "snapremove": rework arguments
- "snapshot": rework argument
- General reworks are based on the .py files.

Make sure to follow and check these boxes before submitting a PR! Thank you.

- [x] Explain the feature
- [x] Read [CONTRIBUTING.md](https://github.com/iocage/iocage/blob/master/CONTRIBUTING.md)

Mrt134 committed:
Initial review of iocage.8 SYNOPSIS

- Added Warren Block and Tim Moore to the author list.
First pass SYNOPSIS review
- Added missing "clone" command.
- "console": remove --force flag.
- "create": add --name flag, remove obsolete commands.
- "df": Add _long, add --sort flags.
- "exec": remove ALL  argument.
- "fetch": add _file, --name, --accept, --noaccept, remove obsolete optional commands.
- "fstab": add ACTION arguments to flags, rework lines.
- "get": rework lines, add _all, add _pool
- "import": remove Jail_value line
- "list": add dataset_type arguments, add --sort flags, add _long flag
- "pkg": remove JAIL arg, replace with UUID | TAG
- "rollback": move jail argument, remove obsolete argument.
- "set": rework  arguments
- "snapremove": rework arguments
- "snapshot": rework argument
- General reworks are based on the .py files.

Mrt134 committed:
Update iocage.8 Authors:

- Adjust authors section to show authors of iocage and authors of the manpage.

Mrt134 committed:
Update to add -l | --long | _long flag to snaplist command:

From this commit: https://github.com/iocage/iocage/commit/211c320c60b6bd278ff36349c5d8a2beb5a79365

Mrt134 committed:
Add more manual page authors:

Add Peter Toth and Brandon Schneider to the manpage author list.

Mrt134 committed:
Merge in manpage changes from #203:

https://github.com/iocage/iocage/commit/acdf1a8737a851a32bd82931246b44e4cf61c3a1
- Moved the added -f flag lines to be alphabetized with the rest of the option flags.

Mrt134 committed:
2nd pass, iocage.8:

Fix PEP8, whitespace errors.

Update iocage.8 DESCRIPTION section:
- Update option flags and descriptions.
- Add missing `clone` subcommand entry.
- remove `-f` flag from console subcommand.
- Add -n, --name flag to create subcommand.
- Add -s, --sort, _sort, and _long flags to df subcommand.
- Add --accept, --noaccept flags to fetch subcommand.
- Rework fetch subcommand.
- Add "action" options to fstab subcommand.
- Add _all and _pool options to get subcommand.
- Add dataset_type, _long, -s | --sort | _sort options to list subcommand.
- Fix option error with list subcommand.
- Rework restart subcommand slightly.
- Add -l, --long, _long options to the snaplist subcommand.
- Rework start subcommand.
- Rework stop subcommand.
- Small rework of upgrade subcommand.

Fix errors in SYNOPSIS:
- Rework rollback entry.
- Fix snapremove entry.
- Fix snapshot entry.
- Fix start entry.
- Fix stop entry.

Mrt134 committed:
Add missing TEXT args to create options in SYNOPSIS and DESCRIPTION.

Mrt134 committed:
Add missing TEXT arg to df SYNOPSIS.

Mrt134 committed:
Merge branch 'master' into patch-1

Mrt134 committed:
Rework iocage fetch examples

- Add basic and more complicated examples.
- Update examples with proper command syntax.

Mrt134 committed:
Add "activate" example to the DESCRIPTION section

Mrt134 committed:
Add more examples to DESCRIPTION section:

- Add simple example of iocage stop.
- Add simple example of iocage chroot.
- Add simple example of iocage clean.
- Add simple example of iocage clone.
- Add simple example of iocage console.
- Rework iocage create examples.
- Add simple examples of iocage destroy.

Mrt134 committed:
Rework iocage create description

Add missing help text.
Rework -c,--count and -b, --basejail option descriptions.
Fix a few errors.
Add another substantive example.

Mrt134 committed:
More examples for the subcommand descriptions:

Expand one destroy example.
Fix an error with df options, add df example.
Add export example.
Add exec examples.
Add simple fstab example (should be replaced with something better).
Add three get examples.
Add simple import example.
Add simple list example with more explanation.
Add simple migrate example.
Add simple restart examples.
Rework set example, add another.
Add simple start example.
Add simple update example.
Rework upgrade example.
Add snapshot example.
Add snaplist example.
Add snapremove example.
Add rollback example.

TODO: add a iocage pkg example (ran into some bumps).

Mrt134 committed:
Addition of iocage list [-q, --quick] flag for #230

Added to SYNOPSIS and DESCRIPTION sections.

Mrt134 committed:
Section reviews:

- Review DESCRIPTION section.
- Fix a few syntax errors.
- Review BUGS section.
- Add VNET to SEE ALSO section.
- Review HINTS section.
- Review EXAMPLES section.

Mrt134 committed:
Issue #173: Update VNET property.

Update the description of the VNET= entry in the PROPERTIES section.
Remove references of VNET autodetection/autoset.

Mrt134 committed:
Review PROPERTIES section

Update syntax to properly show options.
Fix a few errors.
Add some of the optional choices.
Build test and review the entire manual page for any obvious errors.

Mrt134 committed:
Add missing -c, --count description to the fetch subcommand

Mrt134 committed:
Small update to iocage create -t description.

Clarified based on testing.

Mrt134 committed:
Issue #245: Fix ip4, ip6 typo.

Mrt134 committed:
Issue #260: Add more fstab flags

Added -H | -h | --header optional flag to SYNOPSIS and SUBCOMMAND sections.
Added -l | --list optional flag to SYNOPSIS and SUBCOMMAND sections.

skarekrow commented on iocage/iocage.8:
```diff
@@ -24,36 +24,35 @@
+.\"== CLONE ==
+.Nm
+.Cm clone
+.Ar UUID | TAG
```
s/TAG/NAME/g

skarekrow commented on iocage/iocage.8:
```diff
@@ -236,8 +225,8 @@ or just
 .Pp
-To ease jail identification, a TAG field is included in list mode which
-can be set to any string (hostname, label, note, etc.).
+To ease jail identification, a TAG field is included in list mode
```
We use --name now, but this whole bit is now outdated. So remove it, and the date does not get inserted instead, just a plain UUID.

skarekrow commented on iocage/iocage.8:
```diff
@@ -290,90 +279,152 @@ The pool can be activated for
+.Pp
+Example:
+.Pp
+.Dl # iocage clone 38114a58 tag=cloneexample1
```
`--name` instead

skarekrow commented on iocage/iocage.8:
```diff
@@ -290,90 +279,152 @@ The pool can be activated for
+.Pp
+Create a FreeBSD 11.0 jail with the custom UUID 12345678.
+.Pp
+.Dl # iocage create -c 3 -r 11.0-RELEASE tag=examplejail
```
`--name` instead

skarekrow commented on iocage/iocage.8:
```diff
@@ -290,90 +279,152 @@ The pool can be activated for
+This command creates three identical jails based off the
+FreeBSD 11.0 RELEASE.
+These jails are sequentially tagged based on the
+custom tag property.
```
on the name supplied

skarekrow reviewed (changes requested):
Will re-review once the global replacement and such is done.

Mrt134 committed:
Address review comments:

- Replace all listed TAG elements with NAME.
- Remove references to TAG creation and date/time stamps from DESCRIPTION.
- Adjust iocage clone example
- Adjust iocage create example and description.

Mrt134 committed:
Hit commit too soon:

-Replace a few more TAG references.

Mrt134 committed:
Issue #262: change 'set' to show multiple properties

Reworked the 'set' subcommand to show users can designate multiple properties to change in the jail. (SYNOPSIS and SUBCOMMANDS).

skarekrow commented on iocage/iocage.8:
```diff
@@ -774,8 +979,9 @@ interface.
+The IP address is automatically assigned at the first start of the
+jail.
+This requires the
 .Dv ip4_autostart
```
`autostart` and `autoend` don't exist yet. They may or may not return in the same form, so we should nuke references to them.

Mrt134 committed:
Remove autostart and autoend mentions:

PROPERTIES section:
ip4_addr entry
